rock/opium_rock.mli

```diff
@@ -105,5 +105,5 @@ module App : sig
 
   val create : ?middlewares:Middleware.t list -> handler:Handler.t -> t
 
-  val run : t -> port:int -> unit Lwt.t
+  val run : t -> port:int -> ssl:((string * string) option) -> unit Lwt.t
```

Comment:
Can we include the polymorphic variant labels here otherwise it's a bit hard to understand what's being passed.

Also, what about making `~ssl` optional `?ssl` and putting it as the first argument to maintain backwards compat.

Other than that, looks good.